GMImagePicker/GMAlbumsViewController.m

```diff
@@ -123,6 +123,9 @@ - (void)selectAllAlbumsCell {
 
 -(void)updateFetchResults
 {
+    if (self.picker.mediaTypes == NULL) {
+        self.picker.mediaTypes = @[@(PHAssetMediaTypeImage)];
+    }
```

Comment:
I would also recommend checking here if the picker is nil, and if so, just returning early.

Reply:
That's a good point! I will add that.